src/main/java/ch/epfl/biop/wrappers/stardist/Stardist.java

```diff
@@ -73,6 +77,72 @@ static void execute(List<String> options, Consumer<InputStream> outputHandler) t
         System.out.println(cmd.toString().replace(",", ""));
         ProcessBuilder pb = new ProcessBuilder(cmd).redirectErrorStream(true);
 
+*/
+        // start terminal
+        if (IJ.isWindows()) {
+            start_cmd=  Arrays.asList("cmd.exe", "/C");
+        } else if ( IJ.isMacOSX()) {
+            start_cmd = Arrays.asList("bash", "-c");
+        }else if (IJ.isLinux()){
+            throw new UnsupportedOperationException("Linux not supported yet");
+        }
+        cmd.addAll( start_cmd );
+
+
+        // Depending of the env type
+        if (stardistEnvType.equals("conda")) {
+            List<String> conda_activate_cmd = null;
+
+            if (IJ.isWindows()) {
+                // Activate the conda env
```

Comment:
Did you check whether it's still working on windows ?

Reply:
yes